documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -27,10 +28,7 @@ In this procedure:
 . An external listener is configured for the Kafka cluster, with TLS encryption and authentication, and Kafka _simple authorization_ is enabled.
 . A `KafkaUser` is created for the client, with TLS authentication and Access Control Lists (ACLs) defined for _simple authorization_.
 
-You can configure your listener to use TLS or SCRAM-SHA-512 authentication,
-both of which can be used with TLS encryption.
-If you are using an authorization server, you can use token-based link:{BookURLUsing}#assembly-oauth-authentication_str[OAuth 2.0 authentication^] and link:{BookURLUsing}#assembly-oauth-authorization_str[OAuth 2.0 authorization^].
-Open Policy Agent (OPA) authorization is also supported as a link:{BookURLUsing}#con-securing-kafka-authorization-str[Kafka authorization] option.
+You can configure your listener to use TLS or SCRAM-SHA-512 authentication, both of which can be used with TLS encryption.
```

Comment:
I guess this should also mention OAuth authentication? Also, the _both of which can be used with TLS encryption_ sounds weird because TLS authentication has to be used with TLS encryption.It does not work without it.

Reply:
Updated to:

> You can configure your listener to use TLS, SCRAM-SHA-512 or OAuth 2.0 authentication. TLS always uses encryption, but it is recommended to also use encryption with SCRAM-SHA-512 and OAuth 2.0 authentication.
> 
> You can configure simple, OAuth 2.0, OPA or custom authorization for Kafka brokers. When enabled, authorization is applied to all enabled listeners.

Thought I'd add the authorization options too, though we have links in the **.additional resources**.